addon/components/labs-ui/icon-tooltip.js

```diff
@@ -1,14 +1,15 @@
-import Component from '@ember/component';
-import layout from '../../templates/components/labs-ui/icon-tooltip';
+import Component from '@glimmer/component';
 
-export default Component.extend({
-  tagName: 'span',
-  classNames: 'icon-tooltip',
+export default class IconTooltipComponent extends Component {
```

Comment:
Nice. For my own reference:
![image](https://user-images.githubusercontent.com/3311663/77956110-5898d900-729f-11ea-8b33-d7ad1a11d7df.png)
https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/